Marlin/SanityCheck.h

```diff
@@ -1038,3 +1038,21 @@ static_assert(1 >= 0
   #endif
   , "Please select no more than one LCD controller option."
 );
+
+#if ENABLED(HAVE_TMC2130) && \
+    DISABLED(  X_IS_TMC2130 ) && \
+    DISABLED( X2_IS_TMC2130 ) && \
+    DISABLED(  Y_IS_TMC2130 ) && \
+    DISABLED( Y2_IS_TMC2130 ) && \
+    DISABLED(  Z_IS_TMC2130 ) && \
+    DISABLED( Z2_IS_TMC2130 ) && \
+    DISABLED( E0_IS_TMC2130 ) && \
+    DISABLED( E1_IS_TMC2130 ) && \
+    DISABLED( E2_IS_TMC2130 ) && \
+    DISABLED( E3_IS_TMC2130 )
+  #error Choose at least one TMC2130 stepper.
+#endif
+
+#if ENABLED(HYBRID_THRESHOLD) && DISABLED(STEALTHCHOP)
+  #error Enable STEALTHCHOP to use HYBRID_THRESHOLD.
```

Comment:
At some point it was decided to always use quotes on `#error` and `#warning` lines.